package.json

```diff
@@ -31,11 +36,7 @@
     "storybook:build": "cross-env NODE_NO_WARNINGS=1 storybook build --quiet",
     "test:unit": "cross-env NODE_NO_WARNINGS=1 jest --passWithNoTests",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:storybook": "test-storybook --no-index-json",
-    "test:storybook:local": "concurrently -P -r -k -s first npm:storybook \"wait-on http://localhost:6006 && npm run test:storybook -- {@}\"",
-    "test:storybook:snapshot": "npm run test:storybook:local -- -- --updateSnapshot",
-    "test:storybook:watch": "npm run test:storybook:local -- -- --watch",
-    "test": "concurrently -r -s all npm:test:unit npm:test:storybook:local",
+    "test": "npm run test:unit",
```

Comment:
why we nee that simple test:unit work fine ?

Reply:
I didn't get your question.

Reply:
why we add this? Why we don't simple keep `test:unit` ?

Reply:
It's common practice for app projects to have a "test" and "build" commands. This reduces the learning gap if someone wants to test our repository easily.

Reply:
Ok that make sens. +1 to keep that